FilePersistence/src/version_control/Commit.h

```diff
@@ -79,6 +80,8 @@ class FILEPERSISTENCE_API Commit
 		void addFile(QString relativePath, qint64 size, std::unique_ptr<char[], CommitFileContentDeleter> content);
 
 		bool getFileContent(QString fileName, const char*& content, int& contentSize, bool exactFileNameMatching) const;
+		QStringList nodeLinesFromId(Model::NodeIdType id, bool findChildrenByParentId = 0) const;
```

Comment:
do not *ever* use `0` and `1` when working with booleans. Always use `true` and `false`. I know that internally `0` and `1` are the same thing, but this is bad coding practice and is harder to read.